lib/temporal/workflow/serializer/schedule_activity.rb

```diff
@@ -43,7 +43,6 @@ def serialize_retry_policy(retry_policy)
             maximum_interval: retry_policy.max_interval,
             maximum_attempts: retry_policy.max_attempts,
             non_retriable_error_reasons: non_retriable_errors,
-            expiration_interval: retry_policy.expiration_interval
```

Comment:
Hmm, this change is totally unrelated and questionable (https://temporalio.slack.com/archives/CTRCR8RBP/p1603714682311600), can we leave it out of this PR?

Reply:
The field no longer exists, I can split it into another PR if you'd prefer but it needs to be removed.

Reply:
Removed from this PR.